integration-tests/src/test/java/org/openmrs/maven/plugins/SetupIntegrationTest.java

```diff
@@ -215,6 +215,26 @@ public void setup_shouldInstallServerWithGivenJavaHomeAndAddJavaHomeToSdkPropert
         assertThat(server, serverHasDebugPort("1044"));
     }
 
+    @Test
+    public void setup_shouldInstallWithParentDistroSpecifiedInDistroProperties() throws Exception{
```

Comment:
@rkorytkowski I think this can be done once there is at least one release with distro containing parent. We've tested this making one locally.